BGAnimations/ScreenSelectMusic overlay/PerPlayer/DensityGraph.lua

```diff
@@ -0,0 +1,137 @@
+local player = ...
+local pn = ToEnumShortString(player)
+local p = PlayerNumber:Reverse()[player]
+
+local height = 64
+local width = IsUsingWideScreen() and 286 or 276
+
+-- Max height at 280 BPM 16ths. Anything faster will just scale the graph. 
+local maxNps = 280/60 * 4
+
+local af = Def.ActorFrame{
+	InitCommand=function(self)
+		self:visible( GAMESTATE:IsHumanPlayer(player) )
+		self:xy(_screen.cx-182, _screen.cy+23)
+
+		if player == PLAYER_2 then
+			self:addy(height+24)
+		end
```

Comment:
Should we vertically center or scale up the NPS graph when there is only one player?

Pro:
- Use dead space and more symmetry

Contra:
- Consistency. Players get used to always look for the info at the same place.

Reply:
I don't think we should scale up since the NPS graph looks kinda weird the more "square" it gets. I could vertically center, but I think for now I do prefer the consistency of the placements. It also makes the code a bit simpler :)

Reply:
fair